equinix/data_source_fabric_network_acc_test.go

```diff
@@ -10,7 +10,7 @@ import (
 
 func TestAccDataSourceFabricNetwork_PFCR(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		PreCheck:          func() { acceptance.TestAccPreCheck(t); acceptance.TestAccPreCheckProviderConfigured(t) },
```

Comment:
Have you tried running these tests without the added call to `acceptance.TestAccPreCheckProviderConfigured(t)`?  That method was created to fix some tests that cover upgrading from an older version of the provider, because in those tests the first step runs with a pre-built existing release of the provider and that bypasses some necessary test setup.  I wouldn't expect it to be necessary here; is there something about how the Fabric parts of the provider are configured that requires adding this to all Fabric tests?

```suggestion
		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
```

Reply:
With this one, we started getting very random oauth errors when running multiple fabric tests in parallel. The current main branch with `fabric-go` is failing frequently with those errors. When I tried to analyze it this was the only acceptance test change that occurred so I had made the assumption that it was causing it.

I merged in another change to main with this line included in the fabric tests hoping it would resolve the issue but it didn't take. I think with the new auth flow in this implementation the error doesn't persist and this was just to be sure it stayed in and wasn't being accidentally removed in this update. I'll double check and update you on what I find.